proxy/src/serve.rs

```diff
@@ -28,7 +30,10 @@ pub(crate) async fn serve() -> anyhow::Result<()> {
         })
         .expect("Error setting handler for graceful shutdown.");
 
-    info!("Listening for requests on {}", config.bind_addr);
+    let mut api_keys_joined = String::new();
```

Comment:
Please rename `api_keys_joined`, as we are interested in the AppId, not the key.

Reply:
Fixed in 78bd2c8